addons/hide-ads/userscript.js

```diff
@@ -0,0 +1,201 @@
+export default async function ({ addon, console }) {
+  let url;
+  let contentClass;
+  let containerClass;
+  let userClass;
+  let idToContent = {};
+
+  const settingsToUrl = { projects: "projects", studios: "studios", profiles: "users" };
+  const matches = addon.settings.get("matches");
+
+  url = window.location.href;
+
+  if (url.includes("projects")) {
+    contentClass = ".comment-content";
+    containerClass = ".comment-container";
+    userClass = ".username";
+  } else if (url.includes("studios")) {
+    contentClass = ".comment-content";
+    containerClass = ".comment-container";
+    userClass = ".username";
+  } else if (url.includes("users")) {
+    contentClass = ".content";
+    containerClass = ".top-level-reply";
+    userClass = ".name";
+  }
+
+  const advertisingContent = "[Marked as advertising]";
```

Comment:
These should be translated.

Reply:
Oh my god i always forget about translation